src/NuGet.Clients/NuGet.Console/PowerConsoleToolWindow.cs

```diff
@@ -140,14 +140,16 @@ public override void OnToolWindowCreated()
         {
             // Register key bindings to use in the editor
             var windowFrame = (IVsWindowFrame)Frame;
-            Guid cmdUi = VSConstants.GUID_TextEditorFactory;
+            var cmdUi = VSConstants.GUID_TextEditorFactory;
             windowFrame.SetGuidProperty((int)__VSFPROPID.VSFPROPID_InheritKeyBindings, ref cmdUi);
 
+            // commented since this code is only applicable for LSL mode edge case but LSL is being
+            // deprecated
             // refresh PMC UI to update default project name after solution load completed.
-            SolutionManager.SolutionOpened += (o, e) =>
+            /*SolutionManager.SolutionOpened += (o, e) =>
             {
                 VsUIShell.UpdateCommandUI(0);
-            };
+            };*/
```

Comment:
I suggest deleting all of this instead of leaving it commented out, we can find it in git history if needed.